cypress-io/cypress
# fix: hide 'open in IDE' button from command log in CT

- Closes #24366

### User facing changelog


User will no longer see "Open in IDE" in the command log for component tests

### Additional details


This button doesn't work for CT. As a part of the GA effort, we should remove this button from CT so that users don't try to click it. We'll actually fix it and re-enable it with #24549

### Steps to test


1. Run Cypress
2. Open e2e tests
3. Run a spec
4. While the spec is running, hover over the "test body" section of the command log and verify that you still see the "Open in IDE" button on hover
5. Click the button and verify that you are taken to the spec file in your configured IDE
6. Open Component Testing
7. Run a CT spec
8. While the spec is running, hover over the same spot in the command log and verify that the "Open in IDE" button is **not** rendered

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
